### PR TITLE
docs: Added sample env variables for client and server

### LINKED ISSUE #638: [DOCS] add extra variables to env_sample
### Description

add this two to -> https://github.com/Dun-sin/Whisper/blob/main/server/.env_sample
```
MAILTRAP_TOKEN=<mailtraptoken>
MAILTRAP_EMAIL=mailtrap@demomailtrap.com
```

and this to -> https://github.com/Dun-sin/Whisper/blob/main/client/.env_sample
```
VITE_SECRET_KEY=whisper123
```

### Screenshots

_No response_

### Additional information

_No response_

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

None

### PR DESCRIPTION
**Pull request by ChinoUkaegbu**
# Fixes #638 

**My PR closes #638**

# 👨‍💻 Changes proposed
I added the extra sample env variables to both the server and client files

# ✔️ Check List

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.

### CODE REVIEW

**Comment by Dun-sin** [comment]
@ChinoUkaegbu change the title to fit what you did and not what the PR closes, treat it like you're writing a commit message follow this guide(will link them at the end)
but in summary it should be [prefix]: [what you did]

[how to write what you did](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/)

[how to pick which prefix to use](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

**Comment by ChinoUkaegbu** [comment]
Thanks, I've edited the title to reflect that!

File: client/.env_sample [hunk 1]
```diff
@@ -1,2 +1,3 @@
 VITE_IMPORTANT=test-d4a1c830-a65f-4c43-a871-abafcc31066d
 VITE_SOCKET_URL="http://localhost:4000"
+VITE_SECRET_KEY=whisper123
```
Comment (Dun-sin) [inline client/.env_sample:3]:
```suggestion
VITE_SECRET_KEY=whisper123

```

File: server/.env_sample [hunk 2]
```diff
@@ -1,4 +1,6 @@
 THIS IS JUST A FILE TO HELP YOU KNOW WHAT IS NEEDED IN THE .env FILE.
 YOU HAVE TO CREATE ANOTHER FILE AND NAME IT .env
 
-MongoDB_URL=mongodb://username:password@localhost:27018/
\ No newline at end of file
+MongoDB_URL=mongodb://username:password@localhost:27018/
+MAILTRAP_TOKEN=<mailtraptoken>
+MAILTRAP_EMAIL=mailtrap@demomailtrap.com
```
Comment (Dun-sin) [inline server/.env_sample:6]:
```suggestion
MAILTRAP_EMAIL=mailtrap@demomailtrap.com

```

Comment (Dun-sin) [inline server/.env_sample:6]:
```suggestion
MAILTRAP_EMAIL=mailtrap@demomailtrap.com

```

**Review by Dun-sin — APPROVED** [review]
You did great, thanks for contributing, I hope you will stick around and continue to contribute to this project.

Consider giving this project a star, sharing the project with your friends, and joining the community discord server if you haven't for more resources and opportunities to connect with others. [👉🏽here👈🏽](https://discord.com/invite/ufcysW9q23)

**Pull request merged**